src/web/mjs/connectors/Futekiya.mjs

```diff
@@ -0,0 +1,79 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Futekiya extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'futekiya';
+        super.label = 'futekiya';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://read.futekiya.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '.card-body.book-detail h3');
+        let id = uri.pathname;
+        let title = data[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let maxPageCount = 1;
+        let mangas = [];
+        const request = new Request(new URL('/browse?page=1', this.url), this.requestOptions);
+        const newpagecount = await this.fetchDOM(request, ".pagination :nth-last-child(2)");
+        maxPageCount = parseInt(newpagecount[0].innerText.trim());
+        for (let i = 1; i <= maxPageCount; i++) {
+            const request = new Request(new URL('/browse?page=' + i, this.url), this.requestOptions);
```

Comment:
Um, this will cause redeclaration error in runtime. Did you forget to define `request` as `let`?
```suggestion
        let request = new Request(new URL('/browse?page=1', this.url), this.requestOptions);
        const newpagecount = await this.fetchDOM(request, ".pagination :nth-last-child(2)");
        maxPageCount = parseInt(newpagecount[0].innerText.trim());
        for (let i = 1; i <= maxPageCount; i++) {
            request = new Request(new URL('/browse?page=' + i, this.url), this.requestOptions);
```

Reply:
I think it will still work and not cause erors because the const is in a forloop making it on a different level. but it's indeed still a good change.